### ISSUE TITLE
perldata typo

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#1949]
Migrated from [rt.perl.org#3220](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=3220) (status was 'resolved')

Searchable as RT3220$

**Comment by p5pRT** [comment]
### From der.hans@LuftHans.com

moin\, moin\,

&nbsp;       keys\(\) function\.  This rounds up the allocated bucked to
&nbsp;       the next power of two&#8203;:

I presume that should be buckets\, not bucked\.

PERLDATA\(1p\)     Perl Programmers Reference Guide    PERLDATA\(1p\)
29/Dec/1998            perl 5\.005\, patch 03                    13

My operating environment is debian potato for PPC\.

I've also got RedHat 6\.2 on x86 with the same typo\.

2/Feb/2000             perl 5\.005\, patch 03                    13

ans SuSE 6\.something\.

11/Dec/98              perl 5\.005\, patch 02                    13

ciao\,

der\.hans

**Comment by p5pRT** [comment]
### From [Unknown Contact. See original ticket]

This patch fixes the minor doc bug in bug ticket 20000508\.004\.

<details><summary>Inline Patch</summary>

```diff
--- bleadperl.orig/pod/perldata.pod	Thu Jul 27 13:41:29 2000
+++ bleadperl/pod/perldata.pod	Thu Jul 27 13:42:34 2000
@@ -259,7 +259,7 @@
 10,000 of your items.  This isn't supposed to happen.
 
 You can preallocate space for a hash by assigning to the keys() function.
-This rounds up the allocated bucked to the next power of two:
+This rounds up the allocated buckets to the next power of two:
 
     keys(%users) = 1000;		# allocate 1024 buckets
 
```
</details>

**Comment by p5pRT** [comment]
### From @jhi

On Thu\, Jul 27\, 2000 at 01&#8203;:49&#8203;:04PM \+0100\, Michael Stevens wrote&#8203;:
> This patch fixes the minor doc bug in bug ticket 20000508\.004\.
> 
> \-\-\- bleadperl\.orig/pod/perldata\.pod    Thu Jul 27 13&#8203;:41&#8203;:29 2000
> \+\+\+ bleadperl/pod/perldata\.pod    Thu Jul 27 13&#8203;:42&#8203;:34 2000

Thanks\, applied\.

**Comment by p5pRT** [comment]
### From [Unknown Contact. See original ticket]

On Thu\, Jul 27\, 2000 at 04&#8203;:48&#8203;:44PM \+0300\, Jarkko Hietaniemi wrote&#8203;:
> On Thu\, Jul 27\, 2000 at 01&#8203;:49&#8203;:04PM \+0100\, Michael Stevens wrote&#8203;:
> > This patch fixes the minor doc bug in bug ticket 20000508\.004\.
> > \-\-\- bleadperl\.orig/pod/perldata\.pod    Thu Jul 27 13&#8203;:41&#8203;:29 2000
> > \+\+\+ bleadperl/pod/perldata\.pod    Thu Jul 27 13&#8203;:42&#8203;:34 2000
> Thanks\, applied\.

And closed\.

**Comment by p5pRT** [comment]
### From The RT System itself

'patch applied by jarkko\\n'
